src/mount.c

```diff
@@ -874,3 +880,62 @@ cc_get_device_and_fstype(gchar *mount_point, gchar **device_name, gchar **fstype
 	endmntent(file);
 	return ret;
 }
+
+/*!
+ * Check for devicemapper storage for container rootfs.
+ * If devicemapper block device is detected, set the
+ * config->device_name to the device found.
+ *
+ * \param config \ref cc_oci_config.
+ *
+ * \return \c true on success, else \c false.
+ */
+gboolean
+cc_oci_check_for_devicemapper(struct cc_oci_config *config)
```

Comment:
We could make that one a little more generic and call it e.g. `cc_oci_rootfs_is_block_device()`